src/components/KTextArea/KTextArea.vue

```diff
@@ -277,15 +312,44 @@ $kTextAreaPaddingY: var(--kui-space-40, $kui-space-40); // corresponds to mixin,
     }
   }
 
+  .input-wrapper {
+    display: contents;
+
+    &.autosize.legacy {
+      display: grid;
```

Comment:
question: why do we need to change the display here? Couldn't this have downstream impact?

Reply:
https://chriscoyier.net/2023/09/29/css-solves-auto-expanding-textareas-probably-eventually/

This is required by a common hack before `field-sizing: content` is supported.

I'll add the reference link to the comments.

Reply:
There are still a few issues with `display: contents` in Safari; we should make sure they don't have an impact here

Reply:
Or we can simply remove it.

Reply:
Removed `display: contents`.